cypress/integration/Admin.feature

```diff
@@ -25,3 +25,44 @@ Feature: Admin
     And I type "elliot@townx.org" into "email-address-input"
     And I click "submit-button"
     Then deleted user is displayed with deletion date of "5th May 2021 at 12:21:20 pm"
+
+  @focus
+  Scenario: Set and remove a system message on user facing site
+    Given I visit the admin sign-in page
+    And I log in to admin
+    And I click "system-message-link"
+    Then I am taken to the system message page
+
+    # set message to display on front end
+    When I type "Your pizza is burning" into "message"
+    And I click "submit-button"
+    And I see "System message set" in the page text
+
+    # remove message to display on front end
+    Then I clear the value in "message"
+    And I click "submit-button"
+    And I see "System message removed" in the page text
+
+  @focus
+  Scenario: View feedback sent to the service
+    Given I visit the admin sign-in page
+    And I log in to admin
+    And I click "feedback-link"
+    Then I am taken to the feedback page
+
+    # date fields for feedback range
+    Then I fill out "#id-day-start-date" element with "11"
```

Comment:
Any chance of using a data-cy tag for this?  Or did you draw a blank trying to find where to put that?  I had a look myself and its not immediately obvious. ;)

Reply:
It was tried. The place to add it would be either
`service-admin/src/App/templates/app/feedback.phtml`
`service-admin/src/App/src/Form/Feedback.php`
`service-admin/src/App/src/Form/Fieldset/Date.php`

 But it's a field set that's pulled in, trying to applying attributes that aren't on the approved list doesn't work. I looked into trying to add in viewHelpers as we have on `service-front` but there is a fair amount of changes need to do this.

Reply:
Fair enough :)